### ISSUE TITLE
"null byte" error when running bundle in master

### ISSUE THREAD

**Issue by adelevie** [18F/jekyll_pages_api_search#3]
```
➜  jekyll_pages_api_search git:(master) bundle
Fetching gem metadata from https://rubygems.org/...........
Fetching additional metadata from https://rubygems.org/..
Resolving dependencies...
Using rake 10.4.2
Using blankslate 2.1.2.4
Using bundler 1.7.9
Using hitimes 1.2.2
Using timers 4.0.1
Using celluloid 0.16.0
Using fast-stemmer 1.0.2
Using classifier-reborn 2.0.3
Using docile 1.1.5
Using json 1.8.2
Installing simplecov-html 0.10.0
Installing simplecov 0.10.0
Installing codeclimate-test-reporter 0.4.7
Using coffee-script-source 1.9.1.1
Using execjs 2.5.2
Using coffee-script 2.4.1
Using colorator 0.1
Installing unf_ext 0.0.7.1
Using unf 0.1.4
Installing domain_name 0.5.24
Using http-cookie 1.0.2
Installing mime-types 2.6.1
Installing netrc 0.10.3
Installing rest-client 1.8.0
Installing tins 1.5.1
Installing term-ansicolor 1.3.0
Using thor 0.19.1
Installing coveralls 0.8.1
Using ffi 1.9.8
Using hike 1.2.3
Using htmlentities 4.3.3
Using jekyll-coffeescript 1.0.1
Using jekyll-gist 1.2.1
Using jekyll-paginate 1.1.0
Using sass 3.4.14
Using jekyll-sass-converter 1.3.0
Using rb-fsevent 0.9.5
Using rb-inotify 0.9.5
Using listen 2.10.0
Using jekyll-watch 1.2.1
Using kramdown 1.7.0
Using liquid 2.6.2
Using mercenary 0.3.5
Using posix-spawn 0.3.11
Using yajl-ruby 1.2.1
Using pygments.rb 0.6.3
Using redcarpet 3.2.3
Using safe_yaml 1.0.4
Using parslet 1.5.0
Using toml 0.1.2
Using jekyll 2.5.3
Using multi_json 1.11.0
Installing rack 1.6.1
Using tilt 1.4.1
Using sprockets 2.12.3
Installing sprockets-helpers 1.1.0
Installing sprockets-sass 1.3.1
Installing jekyll-assets 0.9.2
Installing jekyll_pages_api 0.1.2
Installing libv8 3.16.14.7
Installing ref 1.0.5
Installing therubyracer 0.12.2

ArgumentError: string contains null byte
An error occurred while installing jekyll_pages_api_search (0.0.0), and Bundler
cannot continue.
Make sure that `gem install jekyll_pages_api_search -v '0.0.0'` succeeds before
bundling.
```

**Comment by mbland** [comment]
Ah, right. It's fixed in 53ac2d3e63eb2a001b010f70e42a09af70252d06 as part of #1, wherein the `'\x0'` string in the gemspec is updated to `"\x0"`. I can push a commit just for that if you prefer.

**Comment by afeld** [comment]
@adelevie Just merged #1...mind trying again?

**Comment by mbland** [comment]
Closed by #1.
